sdk/search/azure-search/src/main/java/com/azure/search/models/ItemWarning.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+package com.azure.search.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents an item-level warning.
+ */
+@Fluent
+public final class ItemWarning {
```

Comment:
ItemWarning vs. IndexerWarning?

Reply:
Not blocking; If you decide to change the name, please ensure the Swagger is updated accordingly.